Sources/RingPublishingTracking/Private/EventsFactory+Audio.swift

```diff
@@ -15,7 +15,7 @@ extension EventsFactory {
         var parameters: [String: AnyHashable] = [:]
         parameters["VE"] = audioEvent.rawValue
         parameters["RT"] = EventType.videoEvent.rawValue
-        parameters["PMU"] = audioMetadata.contentId
+        parameters["PMU"] = audioMetadata.contentId.uuidValidation()
```

Comment:
a jak to da nil to w ogole nie wysle sie pole PMU, tak?

Reply:
właśnie tak